users/api/grpc/endpoint_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-const port = 8080
+const port = 8081
```

Comment:
Why have you changed port?

Reply:
I've changed port because other test uses the same port, so running test concurrently could cause usage of the already taken port.